test/data/test_multi_embedding_tensor.py

```diff
@@ -99,6 +99,39 @@ def test_from_tensor_list():
     assert torch.allclose(met.offset, expected_offset)
     assert_equal(tensor_list, met)
 
+    met_with_nan = met.clone()
+    met_with_nan.values[0::2] = -1
+
+    # Test fill_col
+    met_1 = met_with_nan.clone()
+    for col in range(met.num_cols):
+        met_1.fillna_col(col, 100)
+    assert not torch.all(met_1.values == -1).any()
+
+    # Test fillna with scalar
+    met_2 = met_with_nan.clone()
+    met_2.fillna_(100)
+    assert not torch.all(met_2.values == -1).any()
+
+    # Test fillna with vector of shape (num_cols)
+    met_3 = met_with_nan.clone()
+    met_3.fillna_(torch.tensor([100, 101, 102]))
+    assert torch.all(
+        torch.eq(
+            met_3.values,
+            torch.tensor([
+                [100, 100, 100, 101, 101, 102],
+                [3, 4, 5, 8, 9, 11],
+            ])))
+    assert torch.all(torch.eq(met_3.offset, expected_offset))
```

Comment:
Remove this since it's not used (and the behavior is confusing/unintuitive)

Reply:
Sure